urllib3/connection.py

```diff
@@ -242,14 +242,15 @@ class HTTPSConnection(HTTPConnection):
 
     def __init__(self, host, port=None, key_file=None, cert_file=None,
                  strict=None, timeout=socket._GLOBAL_DEFAULT_TIMEOUT,
-                 ssl_context=None, **kw):
+                 ssl_context=None, servername=None, **kw):
```

Comment:
Can we use the same argument name as `ssl_wrap_socket`?

Reply:
Yep, can do. Changed it to server_hostname everywhere.